docs/reference/api-libraries.md

```diff
@@ -23,6 +23,7 @@
 * {doc}`MIGraphX <amdmigraphx:index>`
 * {doc}`MIOpen <miopen:index>`
 * {doc}`MIVisionX <mivisionx:doxygen/html/index>`
+* {doc}`rocAL <rocal:index>`
 * {doc}`rocDecode <rocdecode:index>`
 * [ROCm Performance Primitives (RPP)](https://rocm.docs.amd.com/projects/rpp/en/latest/)
```

Comment:
Hmm... while you're here, are we able to make this relative link like the others?  or are there issues with that?

Reply:
It's done in what-is-rocm already

Reply:
Done. No issues.